pages/common/tcc.md

```diff
@@ -0,0 +1,16 @@
+# tcc
+
+> A tiny C compiler that can run C source files as scripts and otherwise has command line options similar to `gcc`.
+> More information: <https://bellard.org/tcc/tcc-doc.html>.
+
+- Compile and link `file1.c` and `file2.c` to generate an executable:
+
+`tcc -o {{executable_name}} {{path/to/file1.c}} {{path/to/file2.c}}`
+
+- Directly run `source_file.c` like a script and pass arguments to it:
+
+`tcc -run {{path/to/source_file.c}} {{arguments}}`
+
+- Interpret C source files with a shebang inside the file:
+
+`#!/full/path/to/tcc -run`
```

Comment:
This doesn't match the TLDR syntax.

Reply:
The alternative is to remove this part for not being a terminal command, but surely that's worse than leaving it in as valuable information. I can remove it if you guys really want me to, but I feel that that would be needlessly prohibitive without providing any actual benefit. Especially since this is a useful and unique differentiating feature of `tcc`

Reply:
Ok, sure, we will leave it.